lib/test/Test/Pos/Helpers.hs

```diff
@@ -104,6 +104,15 @@ type IdTestingRequiredClassesAlmost a = (Eq a, Show a, Arbitrary a, Typeable a)
 
 type IdTestingRequiredClasses f a = (Eq a, Show a, Arbitrary a, Typeable a, f a)
 
+identityTestSpec
+  :: forall a. (IdTestingRequiredClassesAlmost a)
+  => (forall b. IdTestingRequiredClassesAlmost b => Spec)
+  -> Spec
+identityTestSpec s = describe (typeName @a) (s @a)
+  where
+    typeName :: forall x. Typeable x => String
```

Comment:
Please make it top-level definition, because now it's copy-pasted from `identityTest`. I know it's small, but still.
Also, I think the definition is wrong, it should be `typeName = show $ typeRep (Proxy @x)` rather than `typeName = show $ typeRep (Proxy @a)` (it works because it's called as `typeName @a`, but it's still wrong conceptually). Can you fix it?